examples/low_level.rs

```diff
@@ -0,0 +1,136 @@
+#![no_std]
```

Comment:
Should we put a disclaimer at the top of this example to mention that you shouldn't really use this crate this way, unless you really have to?

Reply:
Good idea! Thanks